cmd/root.go

```diff
@@ -65,6 +65,11 @@ All Nodes of a k3d cluster are part of the same docker network.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		if flags.version {
 			printVersion()
+		} else {
+			if err := cmd.Usage(); err != nil {
+				log.Errorln(err)
+				os.Exit(1)
```

Comment:
Could as well be `log.Fatalln(err)`,m couldn't it?

Reply:
Well seen. I also replace same occur https://github.com/rancher/k3d/blob/2975be91176fb398333045fcc699c1948e28c53f/cmd/root.go#L81